indico/modules/events/registration/templates/display/conference_home.html

```diff
@@ -33,9 +34,20 @@
                     {%- trans %}Registration{% endtrans %}
                 {%- endif -%}
             </div>
-            {% if user_has_registered %}
-                <div>{% trans %}You have registered for this event.{% endtrans %}</div>
+            {% if user_registrations %}
+                {% if user_registrations < regforms|length %}
+                    <div>{% trans %} You have registered for this event, but there are more registration forms available.{% endtrans %}</div>
+                {% else %}
+                    <div>{% trans %} You have registered for this event.{% endtrans %}</div>
+                {%  endif %}
                 <div class="toolbar">
+                    {% if is_number_published %}
+                        <span class="i-label icon-user" title="{% trans %}Total registrations{% endtrans %}">{{ regforms[0].active_registrations|length }}
```

Comment:
I'd end the line at `>` and continue in the next one. We usually only use `<foo>bar` if `</foo>` is in the same line